ngraph/test/op_eval/unary_ops.cpp

```diff
@@ -0,0 +1,186 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <string>
```

Comment:
I don't want to merge several operations to one file. Can we use the same approach with parameterized tests as we use in visitor tests? Each operation has separate test file.

Moreover I dislike op_eval tests which test checks that evaluate method works as expected. But we moved INTERPRETER backend to evaluate methods, it means that we can add only backend tests instead op op_eval.

At the current moment I propose to add only backend tests for new operations and do not add new op_eval tests.